flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -52,7 +52,74 @@
 		key("yarn.per-job-cluster.include-user-jar")
 			.defaultValue("ORDER");
 
+	/**
+	 * The vcores exposed by YARN.
+	 */
+	public static final ConfigOption<Integer> YARN_VCORES =
+		key("yarn.containers.vcores")
+		.defaultValue(Integer.MAX_VALUE);
```

Comment:
is this really a reasonable default value? I wouldn't be surprised if YARN would reject this.